config/relayer/config.go

```diff
@@ -46,6 +47,10 @@ type TopologyConfiguration struct {
 	Path          string `mapstructure:"Path" json:"path"`
 }
 
+type UploaderConfig struct {
+	URL   string `mapstructure:"url"`
+	Token string `mapstructure:"token"`
```

Comment:
Can we rename this to something like `AuthToken`? Honestly, I wasn't sure if you were referring to some on-chain token until I checked out the rest of the code.